spec.yaml

```diff
@@ -1246,6 +1247,45 @@ paths:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/organizations/{organization_id}/credentials/:
+    get:
+      operationId: getCredentials
+      tags:
+        - organizations
+      summary: Get credentials
+      description: |
+        Returns credentials for an organization, if available.
+        For more information on credentials,
+        see [Set credentials](#operation/addCredentials).
+      parameters:
+        - $ref: './parameters.yaml#/parameters/RequiredGiantSwarmAuthorizationHeader'
+        - $ref: './parameters.yaml#/parameters/XRequestIDHeader'
+        - $ref: './parameters.yaml#/parameters/XGiantSwarmActivityHeader'
+        - $ref: './parameters.yaml#/parameters/XGiantSwarmCmdLineHeader'
+        - $ref: './parameters.yaml#/parameters/OrganizationIdPathParameter'
+      responses:
+        "200":
+          description: Credentials
+          schema:
+            $ref: './definitions.yaml#/definitions/V4GetCredentialsResponse'
+          examples:
+            application/json:
+              [
+                {
+                  "id": "a1b2c3",
+                  "provider": "azure",
```

Comment:
can we have an example response for aws, too?

Reply:
We can have only one example response (per content type, to be exact). Unfortunately. But I can add eamples to the description text.

Reply:
Added to the description